LWMS ID 5761: Added Event folder to use Markdown file:  event-register-free-tier-account.md

plforacle opened the pull request:
LWMS ID 5761: Added Event folder to use Markdown file:  event-register-free-tier-account.md

madhusudhanrao-ppm commented:
can you please change this USE MySQL HEATWAVE FOR DEVELOPMENT to remove all capital letters both in manifest.json and MD, other changes look good

plforacle commented:
LWMS ID 5761: Removed all capital letters both in manifest.json and MD

plforacle commented:
Madhusudhan,
I have made the changes and updated GitHub. The workshop is scheduled for 12:30 pm ET today. Is it possible to expedite the production deployment process?

Thank you,
Perside Foster | MySQL Senior Partners Solution Engineer
email: ***@***.******@***.***>
phone: 1.786.877.9063
Oracle  Florida

From: Madhusudhan Rao ***@***.***>
Sent: Thursday, March 17, 2022 1:13 AM
To: oracle/learning-library ***@***.***>
Cc: Perside Foster ***@***.***>; Author ***@***.***>
Subject: [External] : Re: [oracle/learning-library] LWMS ID 5761: Added Event folder to use Markdown file: event-register-free-tier-account.md (PR #4705)


can you please change this USE MySQL HEATWAVE FOR DEVELOPMENT to remove all capital letters both in manifest.json and MD, other changes look good

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/oracle/learning-library/pull/4705*issuecomment-1070333806__;Iw!!ACWV5N9M2RV99hQ!b9htffhL0wjaPMe4wvK2ZTvBgXZ3apLydFJLoxQaubEOtxTfpT9C7CgKHcLbE60laII$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/AMYQBSVDXVRH7AB5URNQ2XDVAK5PJANCNFSM5Q44IHQQ__;!!ACWV5N9M2RV99hQ!b9htffhL0wjaPMe4wvK2ZTvBgXZ3apLydFJLoxQaubEOtxTfpT9C7CgKHcLbH563wFc$>.
Triage notifications on the go with GitHub Mobile for iOS<https://urldefense.com/v3/__https:/apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675__;!!ACWV5N9M2RV99hQ!b9htffhL0wjaPMe4wvK2ZTvBgXZ3apLydFJLoxQaubEOtxTfpT9C7CgKHcLbwLHjK3Y$> or Android<https://urldefense.com/v3/__https:/play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign*3Dnotification-email*26utm_medium*3Demail*26utm_source*3Dgithub__;JSUlJSU!!ACWV5N9M2RV99hQ!b9htffhL0wjaPMe4wvK2ZTvBgXZ3apLydFJLoxQaubEOtxTfpT9C7CgKHcLb2Yd7D_A$>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>
